src/scripts/content/todoist.js

```diff
@@ -34,25 +35,68 @@ togglbutton.render('.task_item .content:not(.toggl)', {observe: true}, function
     return clone.textContent.trim();
   };
 
-  projectFunc = function () {
-    var projectElem, projectLabel;
-    projectElem = $('.project_link');
-    if (projectElem) {
-      return projectElem.textContent.trim();
+  getSidebarEle = function (elem) {
+    var editorInstance, projectId, sidebarRoot, sidebarEle;
+    editorInstance = elem.closest('.project_editor_instance');
```

Comment:
You were probably looking at the Inbox screen, which I realized I hadn't handled. Items in the Inbox should now have the Toggl button link appearing next to them.